src/components/Heading/index.js

```diff
@@ -0,0 +1,10 @@
+import React, { Component } from "react";
+import styled from "styled-components";
+
+const Heading = styled.h1`
+  font-size: ${props => props.theme.font.size.heading.primary};
```

Comment:
well personaly, I'm convinced to use css/scss 🙃

Reply:
The only thing I don't like is how long the code is, unfortunately. :(